app-config.yaml

```diff
@@ -0,0 +1,21 @@
+# ref https://raw.githubusercontent.com/nais/naisd/master/nais_example.yaml
+image: docker.adeo.no:5000/fo/pinpoint
+replicas:
+  min: 2
+  max: 4
+  cpuThresholdPercentage: 50
+port: 8080
+healthcheck:
+  liveness:
+    path: pinpoint/internal/isAlive
+    initialDelay: 10
```

Comment:
egentlig kan du bare fjerne denne linja og bruke default, selv om den trolig er mer enn 10 sekunder

Reply:
Fjerna `initialDelay` for både `liveness` og `readiness`